.eslintignore

```diff
@@ -2,6 +2,5 @@
 /packages/**/node_modules
 /packages/node_modules
```

Comment:
I think that `/packages/**/node_modules` already covers `/packages/node_modules`

Reply:
I assume I added it because it did not. 🤷